deps/build.jl

```diff
@@ -1,15 +1,48 @@
 #######################################################################
-import JSON
+import JSON, Conda
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
 # Make sure Python uses UTF-8 output for Unicode paths
 ENV["PYTHONIOENCODING"] = "UTF-8"
 
-include("jupyter.jl")
-const jupyter, jupyter_vers = find_jupyter()
+
+const jupyter, jupyter_vers = try
+    "jupyter",convert(VersionNumber, chomp(readall(`jupyter-kernelspec --version`)))
+catch e1
+    try
+        "ipython",convert(VersionNumber, chomp(readall(`ipython --version`)))
+    catch e2
+        try
+            "ipython2",convert(VersionNumber, chomp(readall(`ipython2 --version`)))
+        catch e3
+            try
+                "ipython3",convert(VersionNumber, chomp(readall(`ipython3 --version`)))
+            catch e4
+                try
+                    "ipython.bat",convert(VersionNumber, chomp(readall(`ipython.bat --version`)))
+                catch e5
+                    eprintln("No system wide Jupyter or IPython is found, got the following errors\n",
```

Comment:
system wide -> system-wide
is found -> was found
, got -> ; got
errors -> errors: